.github/workflows/pull-request.yaml

```diff
@@ -53,4 +56,78 @@ jobs:
         with:
           go-version: '1.20'
       - name: Run tests
-        run: make test
\ No newline at end of file
+        run: make test
+
+  changes:
+    runs-on: ubuntu-latest
+    # Required permissions
+    permissions:
+      pull-requests: read
+    # Set job outputs to values from filter step
+    outputs:
+      change: ${{ steps.filter.outputs.api == 'true' || steps.filter.outputs.cmd == 'true' || steps.filter.outputs.config == 'true' || steps.filter.outputs.controllers == 'true' || steps.filter.outputs.internal == 'true' || steps.filter.outputs.manifests == 'true' || steps.filter.outputs.pkg == 'true' || steps.filter.outputs.main == 'true' || steps.filter.outputs.mod == 'true' }}
+    steps:
+      - uses: dorny/paths-filter@4512585405083f25c027a35db413c2b3b9006d50 # v2.11.1
```

Comment:
I don't know if we should use a separate 3rd party action for this. I think we can achieve the same when we create a new workflow that runs on PRs and uses [path filter](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-including-paths).

Reply:
Yeah good idea. I was looking for this in GH documentation but couldn't find :)

Reply:
https://github.com/kyma-project/istio/pull/210/commits/c5a78ac61fa296ad7748674fe153c471a794cd0d